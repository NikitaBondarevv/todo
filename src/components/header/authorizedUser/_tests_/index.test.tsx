import { render } from '@testing-library/react'

import { AuthorizedUser } from '..'

jest.mock('components/header/authorizedUser', () => ({
  AuthorizedUser: () => <div />
}))

describe('<Header />', () => {
  test('should render div', () => {
    const { asFragment } = render(
      <AuthorizedUser />
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
