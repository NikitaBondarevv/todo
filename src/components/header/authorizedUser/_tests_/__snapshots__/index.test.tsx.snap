// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render div 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
